### PR TITLE
Tickets/dm 32251

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: python/lsst/ts/ess/common/test_utils.py [hunk 1]
```diff
@@ -100,6 +105,14 @@ def check_hx85ba_reply(
                     assert common.device.MockPressureConfig.min <= resp[i]
                     assert resp[i] <= common.device.MockPressureConfig.max
 
+        dew_point = common.sensor.Hx85baSensor.compute_dew_point(
+            relative_humidity=resp[0], temperature=resp[1]
+        )
+        if math.isnan(resp[0]) or math.isnan(resp[1]):
+            assert math.isnan(resp[3])
+        else:
+            assert resp[3] == pytest.approx(dew_point)
```
Comment (r-owen) [inline python/lsst/ts/ess/common/test_utils.py:114]: Note that pytest.approx has a `nan_ok` float, so I think you can just write:
```
assert resp[3] == pytest.approx(dew_point, nan_ok=True)
```

**Review by r-owen — APPROVED** [review]
This looks good. I do have one comment about rounding the dew point that I hope you will consider before merging (as well as one or two trivial things).

File: python/lsst/ts/ess/common/abstract_command_handler.py [hunk 2]
```diff
@@ -221,18 +221,18 @@ def create_device(
...
         Returns
         -------
-        device : common.device.BaseDevice`
+        device : `common.device.BaseDevice`
             The device to connect to.
 
         Raises
         ------
-        RuntimeError
+        `RuntimeError`
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/ess/common/abstract_command_handler.py:235]: It does.

File: python/lsst/ts/ess/common/sensor/omega_hx85ba.py [hunk 3]
```diff
@@ -74,17 +87,44 @@ def __init__(
...
+
+        Returns
+        -------
+        `float`
+            Dew point (C)
+        """
+        # Saftey net to ensure the correct return type because `round` doesn't
+        # preserve the type of nan properly.
+        if math.isnan(relative_humidity) or math.isnan(temperature):
+            return math.nan
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/ess/common/sensor/omega_hx85ba.py:109]: In the unit tests comparing lists of NaN with assert fails if I don't include the first lines. But I see your point and have dropped the rounding and therefore those NaN lines as well.

File: python/lsst/ts/ess/common/test_utils.py (hunk 1 above)
Comment (wvreeven) [reply python/lsst/ts/ess/common/test_utils.py:114]: Works wonderfully well, thanks!

**Pull request merged**